tests/acceptance/features/bootstrap/SpacesContext.php

```diff
@@ -2414,4 +2528,165 @@ public function listFileVersion(
 		$responseXml = HttpRequestHelper::getResponseXml($this->featureContext->getResponse());
 		return $responseXml->xpath("//d:response/d:href");
 	}
+
+	/**
+	 * @throws GuzzleException
+	 */
+	public function userGetsEtagOfElementInASpace($user, $space, $path) {
```

Comment:
a good description can be commented regarding what this  function actually does.